assets/src/components/nav/leftNav.tsx

```diff
@@ -26,13 +26,9 @@ import { LinkData, getNavLinkData } from "../../navLinkData"
 
 interface LeftNavLinkProps {
   linkData: LinkData
-  collapsed: boolean
 }
 
-const LeftNavLink = ({
-  linkData,
-  collapsed,
-}: LeftNavLinkProps): JSX.Element => (
+const LeftNavLink = ({ linkData }: LeftNavLinkProps): JSX.Element => (
```

Comment:
praise: I love the removal of the `collapsed` prop!

Glad it also removed some conditionals and allows the buttons to still have names in the accessibility tree :)